src/compiler/scala/tools/nsc/javac/JavaParsers.scala

```diff
@@ -800,16 +800,8 @@ trait JavaParsers extends ast.parser.ParsersCommon with JavaScanners {
       val superclazz =
         AppliedTypeTree(javaLangDot(tpnme.Enum), List(enumType))
       val finalFlag = if (enumIsFinal) Flags.FINAL else 0l
-      val abstractFlag = {
-        // javac adds `ACC_ABSTRACT` to enum classes with deferred members
-        val hasAbstractMember = body exists {
-          case d: DefDef => d.mods.isDeferred
-          case _         => false
-        }
-        if (hasAbstractMember) Flags.ABSTRACT else 0l
-      }
```

Comment:
Why remove this logic? @lrytz added this in https://github.com/scala/scala/commit/63812c18b23de60039fd3267e4806449ea679972#diff-226e876495a438b9b5c6d407d5997368R801